### PR TITLE
Toastify

### PR DESCRIPTION
**Pull request by lodarandrea**
resolving issue #7

### CODE REVIEW

File: src/components/CartItem.tsx [hunk 1]
```diff
@@ -2,18 +2,18 @@ import {
...
 import DeleteRoundedIcon from '@mui/icons-material/DeleteRounded'
 import { useAppDispatch } from '../store/Hooks'
 
 export interface CardItemPops {
-  item: CartItems
+  item: CartItem
 }
 
-function CartItem({ item }: CardItemPops) {
+function CartItems({ item }: CardItemPops) {
```
Comment (kbertalan) [inline src/components/CartItem.tsx:16]: This rename will conflict with another PR.

File: src/pages/Menu.tsx [hunk 2]
```diff
@@ -3,13 +3,24 @@ import { customFetch } from '../services/FetchService'
...
+import { toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
 
 function Menu() {
   const [menuItems, setMenuItems] = useState<Array<MenuItem>>([])
+  const [fetchError, setFetchError] = useState('')
+  const toastId = 'toastCustomId'
   useEffect(() => {
-    customFetch('/food', setMenuItems)
+    customFetch('/fod', setMenuItems, setFetchError)
```
Comment (kbertalan) [inline src/pages/Menu.tsx:14]: `/fod` ?

File: src/pages/Menu.tsx [hunk 3]
```diff
@@ -3,13 +3,24 @@ import { customFetch } from '../services/FetchService'
...
   }, [])
 
+  if (fetchError) {
```
Comment (kbertalan) [inline src/pages/Menu.tsx:17]: It seems the `useState` for `fetchError` is overkill.

**Review by kbertalan — CHANGES_REQUESTED** [review]

File: src/components/CartItem.tsx (hunk 1 above)
Comment (lodarandrea) [reply src/components/CartItem.tsx:16]: done in another PR

File: src/pages/Menu.tsx (hunk 2 above)
Comment (lodarandrea) [reply src/pages/Menu.tsx:14]: changed

**Comment by lodarandrea** [comment]
@kbertalan

**Review by kbertalan — APPROVED** [review]

**Pull request merged**